tests/testutil/service.go

```diff
@@ -263,6 +264,30 @@ func (ts *TestService) GetKubeEndpointURL() string {
 	return strings.Replace(url, "\"", "", -1) // remove double quotes from output
 }
 
+// GetKubeAvailableReplicas returns the app number of ready replicasets with kubectl
+// and returns it as integer
+func (ts *TestService) GetKubeAvailableReplicas() int {
```

Comment:
This whole approach still feels odd, we should chat during standup, I'm still not convinced we need duplicated kube testing. If we do continue this route, instead of using kubectl can we use the kube clientset ? 
https://github.com/kubernetes/kubernetes/blob/master/test/integration/deployment/util.go#L61
https://github.com/kubernetes/kubernetes/blob/master/test/integration/deployment/util.go#L447-L453

Reply:
Uhm I understand your suggestion @cbron , use something like this https://github.com/kubernetes/client-go instead of invoking kubectl from command line? we should meet with @sangeethah about this, I'm open to follow that path if everyone is ok with it.

Reply:
That would likely be better